src/pages/scheduler/components/share/SelectedCoursesCardList.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Flex, Heading, HStack, VStack, Wrap, WrapItem, Text } from '@chakra-ui/react';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+import { SavedCourse } from 'lib/hooks/useSavedCourses';
+import { getReadableTerm } from 'lib/utils';
+
+type ShareCourseCard_Props = {
+  course: SavedCourse;
+};
+
+const ShareCourseCard = ({ course }: ShareCourseCard_Props) => {
+  const mode = useDarkMode();
+
+  return (
+    <Flex height="100%" direction="column">
+      <HStack w="100%" bg={mode('gray.100', 'gray.600')} justifyContent="center" p="0.3em" borderTopRadius="4px">
+        <Heading size="xs">
+          {course.subject} {course.code}
```

Comment:
If you are never explicitly using `course` and only it's attributes, it's probably better to define `subject, code, color, lecture, lab, and tutorial` as separate props OR you can do object de-structuring on course